src/pages/registration/new_registration/LinkRegistration.tsx

```diff
@@ -76,8 +135,59 @@ export const LinkRegistration = ({ expanded, onChange }: StartRegistrationAccord
       </AccordionSummary>
 
       <AccordionDetails>
-        <LinkRegistrationForm handleSearch={handleSearch} />
+        <Formik onSubmit={onSubmit} initialValues={emptyDoiFormValues} validationSchema={doiValidationSchema}>
+          {({ isSubmitting }) => (
+            <Form noValidate>
+              <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                <Field name={LinkRegistrationFormFieldName.Link}>
+                  {({ field, meta: { error, touched } }: FieldProps<string>) => (
+                    <TextField
+                      sx={{ mr: '1rem' }}
+                      id={field.name}
+                      data-testid="new-registration-link-field"
+                      variant="filled"
+                      label={t('registration.registration.link_to_resource')}
+                      required
+                      fullWidth
+                      disabled={isSubmitting}
+                      {...field}
+                      error={!!error && touched}
+                      InputLabelProps={{ shrink: true }}
+                      placeholder={doiUrlPlaceholder}
+                    />
+                  )}
+                </Field>
+                <LoadingButton
+                  data-testid="doi-search-button"
+                  variant="contained"
+                  loading={isSubmitting}
+                  type="submit"
+                  endIcon={<SearchIcon />}
+                  loadingPosition="end">
+                  {t('common.search')}
+                </LoadingButton>
+              </Box>
+            </Form>
+          )}
+        </Formik>
         {noHit && <Typography sx={{ mt: '1rem' }}>{t('common.no_hits')}</Typography>}
+        {searchResults.length > 0 && (
```

Comment:
Mulig denne skulle blitt endret, så søketreffet ikke dukker opp lenge før doi-fetch er ferdig.

![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/53341294/88d4a723-de34-4a05-b88b-4b1992c66dca)

Reply:
om du ikke alt har gjort det burde kanskje flyten her demonstreres for terje/jan erik/cecilie før vi går videre for å være sikre på at det er et steg i riktig retning? ikke minst for å være åpen på at det fortsatt vil opprettes en registrering ved doi-lookup selv om vi finner en duplikat (https://unit.atlassian.net/browse/NP-2958). ellers tror jeg de fort forventer at vi har løst mer enn vi har 😬 
jeg har ikke kontroll på hva som er logisk og ønsket her, egentlig :/

Reply:
^ bump :)
om det er gjort ser det greit ut for meg

Reply:
Verdt å merke at når vi søker for alle felter vil det være tilfeller hvor infomelding ikke stemmer også, feks om man har nevnt en doi/URL i beskrivelsen, så betyr ikke det at det er samme resultat. På bildet her har nr1 bare nevnt samme DOI i description, og er (sannsynligvis) ikke en representasjon av det samme resultatet

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/3392571f-671a-4413-ba93-fd034bbbc249)

Reply:
Dette har blitt løst vha nytt endepunkt som søker eksplisitt på doi